### PR TITLE
Feature/basic setup

### PR DESCRIPTION
**Pull request by hoomberto**
# Description
Add basic setup for backend API
- Add cors, express, and pg dependencies
- Add nodemon devdependency
- Add Pool configuration to handle concurrent connections to database
- Add get requests for all questions
- Add database.sql for basic structure of tables

Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactor

# How Has This Been Tested?
Install dependencies, run nodemon and visit localhost:3001/questions (after setting up database)

Need to update GET requests with URL to reflect that this is an API ('/api/v1/questions')

### CODE REVIEW

File: database.sql [hunk 1]
```diff
@@ -0,0 +1,66 @@
+CREATE DATABASE betterjeopardy;
+
+CREATE TABLE question(
```
Comment (novaraptur) [inline database.sql:3]: Good reference for formatting on our ends

File: database.sql [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+);
+
+
+
+CREATE TABLE categories(
+    category_id SERIAL PRIMARY KEY,
+    name text NOT NULL
+);
+
+ INSERT INTO question (question_id, question, correct_answer, wrong_answer1, wrong_answer2, wrong_answer3, prize)
```
Comment (novaraptur) [inline database.sql:31]: Also good formatting for any future POST requests

**Review by novaraptur — APPROVED** [review]
Nice job! This makes PostgresQL look a lot less intimidating

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (shawnmcmahon) [inline .gitignore:1]: Good call adding this!

File: database.sql [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+CREATE TABLE question(
+    question_id SERIAL PRIMARY KEY,
+    question text NOT NULL,
+    correct_answer text NOT NULL,
+    wrong_answer1 text NOT NULL,
+    wrong_answer2 text NOT NULL,
+    wrong_answer3 text NOT NULL,
+    prize INT NOT NULL,
+    FOREIGN KEY (categoryID) REFERENCES categories (category_id)
```
Comment (shawnmcmahon) [inline database.sql:11]: We will need to do some research into the FOREIGN KEY functionality. Great work setting up this database.

**Pull request merged**